src/Web/Grand.Web/Views/PdfTemplates/OrderPdfTemplate.cshtml

```diff
@@ -8,11 +8,12 @@
 @inject Grand.Business.Core.Interfaces.Checkout.Payments.IPaymentService _paymentService
 @inject Grand.Business.Core.Interfaces.Common.Directory.ICurrencyService _currencyService
 @inject Grand.Business.Core.Interfaces.Checkout.GiftVouchers.IGiftVoucherService _giftVoucherService
-
+@inject IWorkContext workContext
 @{
     var logoPicture = await _pictureService.GetPictureById(_pdfSettings.LogoPictureId);
     var logoExists = logoPicture != null;
-    var limitbyVendor = !string.IsNullOrEmpty(Model.Item2);
+    var limitbyVendor = !string.IsNullOrEmpty(Model.Item2); 
+    var priceIncludesTax = workContext.TaxDisplayType == TaxDisplayType.IncludingTax;
```

Comment:
TaxDisplayType should get from the Order

Reply:
@KrzysztofPajak updated code as per feedback